docs/docs/articles/run-tests-with-github-actions.md

```diff
@@ -0,0 +1,143 @@
+# Run Tests with GitHub Actions
+
+**Run on Testkube** is a GitHub Action for running tests on the Testkube platform.
+
+Use it to run tests and test suites and obtain results directly in the GitHub's workflow.
+
+## Usage
+To use the action in your GitHub workflow, use kubeshop/testkube-run-action@v1 action. The configuration options are described in the Inputs section and may vary depending on the Testkube solution you are using (cloud or self-hosted) and your needs.
+
+The most important options you will need are test and testSuite - you should pass a test or test suite name there.
+
+### Testkube Cloud
+To use this GitHub Action for the Testkube Cloud, you need to create an API token.
+
+Then, pass the **organization** and **environment** IDs for the test, along with the **token** and other parameters specific for your use case:
+
+```sh
```

Comment:
Does our docs support `yaml` highlighting? If so, we could use `yaml` both here and in the rest of snippets:

```suggestion
```yaml
```

Reply:
@aabedraba Can you answer this question? Thanks!

Reply:
we do support yaml, @rangoo94 
so yes, please use yaml for yaml snippets

Reply:
Updated